groups/prometheus/module-ec2-instances/variables.tf

```diff
@@ -0,0 +1,48 @@
+# Instance
```

Comment:
Remove the comment here and others from the remainder of this file, and instead order the variable declarations throughout the file alphabetically based on the variable name.

Although the intention of the comments is good—to group variables by use-case—they've proved to be ambiguous before now and if instead we use consistent variable names, with consistent prefixes, things will naturally be grouped together and we aren't forced to try and maintain order based on a set of arbitrary comments.